jellyfin_kodi/jellyfin/websocket.py

```diff
@@ -545,8 +545,8 @@ def _validate_header(self, headers, key):
             return False
         result = result.lower()
 
-        value = key + "258EAFA5-E914-47DA-95CA-C5AB0DC85B11"
-        hashed = base64.encodestring(hashlib.sha1(value).digest()).strip().lower()
+        value = key + "258EAFA5-E914-47DA-95CA-C5AB0DC85B11".encode()
```

Comment:
What exactly is this string for?

Reply:
Hippy space magic?  At this point i genuinely don't know.

It's generating a uuid to use as a key (line 236), then combining that key with this string and hashing it, and somehow that's equivalent to what the websocket returns when the authentication/handshake process is successful (lines 516-527).  I'm not going to pretend to know how that string was found at this point.